app/src/main/java/com/nightscout/android/SyncingService.java

```diff
@@ -221,8 +221,6 @@ protected void handleActionSync(int numOfPages, Context context, DeviceTransport
                         results.getDisplayTime(), results.getResultArray(), download.receiver_battery,
                         builder.build().toByteArray(), lastSgvTimestamp, lastMeterTimestamp, lastSensorTimestamp, lastCalTimestamp);
                 broadcastSent = true;
-                reporter.report(EventType.DEVICE, EventSeverity.INFO,
```

Comment:
Good catch. I think I mistakenly removed it. I definitely think it would be useful to have this. I'll re-add it this evening.